conformance/metadata-fhirr4-service/README.md

```diff
@@ -1 +1,112 @@
 # FHIR R4 Metadata Service
+
+## Introduction
+This service provides implementation of FHIR Metadata API. This implements 
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+```Supported FHIR version is 4.0.1.```
+
+### Setup and run
+
+1. Clone this repository to your local machine and navigate to the pre-built service on metadata-fhirr4-service.
+
+2. Configure the Config.toml with relevant configurations mentioned in [Configurations](#configurations).
+
+    A sample `Config.toml` is as below.
+
+    ```
+    ## server related configurables
+    [configFHIRServer]
+    version = "1.2.0"
+    name = "WSO2OpenHealthcareFHIR"
+    title = "FHIR Server"
+    status = "active"
+    experimental = true
+    date = "2022-11-24"
+    kind = "instance"
+    fhirVersion = "4.0.1"
+    format = ["json"]
+    patchFormat = ["application/json-patch+json"]
+    implementationUrl = "<FHIR_BASE_URL>"
+    implementationDescription = "WSO2 Open Healthcare FHIR"
+
+    ## server security related configurables
+    [configRest]
+    mode = "server"
+    resourceFilePath = "resources/fhir_resources.json"
+    interaction = ["search-system"]
+    [configRest.security]
+    cors = false
+    discoveryEndpoint = "https://api.asgardeo.io/t/<organization_name>/oauth2/token/.well-known/openid-configuration"
+    managementEndpoint = "https://api.asgardeo.io/t/<organization_name>/oauth2/manage"
+    ```
+
+3. Run the project.
+
+    ```ballerina
+    bal run
+    ```
+
+4. Invoke the APIs.
```

Comment:
```suggestion
4. Invoke the API.
```

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/3c45d53b694704102f3950d7f4431d600fc9e72e